lib/active_merchant/billing/gateways/redsys_rest.rb

```diff
@@ -465,9 +465,14 @@ def validate_signature(data, signature, order_number)
         Base64.urlsafe_encode64(mac256(key, data)) == signature
       end
 
-      def authorization_from(response)
-        # Need to get updated for 3DS support
-        [response[:ds_order], response[:ds_amount], response[:ds_currency]].join('|')
+      def authorization_from(response, post, options)
+        # Needed update for 3DS support
```

Comment:
Does this comment mean that it still needs an update? Can the comment be deleted?